.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,156 @@
+
+      **Logs and network traces**
+      Without logs or traces, it is unlikely that the team can investigate your issue. Capturing logs and network traces is described in our [Docs](https://learn.microsoft.com/azure/active-directory/develop/msal-logging-dotnet).
+
+- type: markdown
+  attributes:
+    value: |
+      ## Issue details   
+    
+- type: input
+  attributes:
+    label: Library version used
+    description: "Please enter the version of the library where you ran into the issue (e.g. 4.54.0)."
+  validations:
+    required: true
+    
+- type: textarea
+  attributes:
+    label: .NET version
+    description: "Please enter the .NET platform version your application was developed in."
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: MSAL client type
+    description: "Are you using PublicClientApplication,  ConfidentialClientApplication or ManagedIdentityApplication?"
+    multiple: true
+    options: 
+      - "Public"
+      - "Confidential"
+      - "Managed identity"
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Application type
+    description: "Please select your application type"
+    multiple: true
+    options: 
+      - "Desktop"
+      - "Mobile"
+      - "Web app"
+      - "Web API"
+      - "Daemon app" 
+  validations:
+    required: true   
+
+- type: dropdown
+  attributes:
+    label: Authentication flow type
+    description: "What authentication flow was used when the issue appeared?"
+    multiple: true
+    options: 
+      - "Interactive (with an authentication broker)"
+      - "Interactive (with embedded or system browser)"
+      - "Integrated Windows Authentication"
+      - "Username and password (ROPC)"
+      - "Device code (browserless)"
+      - "Authorization code"
+      - "On-Behalf-Of"
+      - "Client credentials (service-to-service calls)"
+      - "Long-running On-Behalf-Of (OBO)"
+      - "Managed Identity"
+      - "Other"
+  validations:
+    required: true    
+
+- type: dropdown
+  attributes:
+    label: Is this a new or an existing app?
+    description: "Is this a new or existing app?"
+    multiple: false
+    options: 
+      - "The app is in production, and I have upgraded to a new version of MSAL"
+      - "The app is in production, I haven't upgraded MSAL, but started seeing this issue"
+      - "This is a new app or experiment"
+  validations:
+    required: true        
+
+- type: textarea
+  attributes:
+    label: Issue description and reproduction steps
+    description: "Briefly explain the issue you are seeing. Provide steps to reproduce or a link to a sample which demonstrates the behavior."
+  validations:
+    required: true      
+    
+- type: textarea
+  attributes:
+    label: Error message
```

Comment:
merge with "Issue description"? 

(this is personal feedback from filling in the MSAL JS bug template some 10 times already. They have so many textboxes which are mandatory and which I never fill in)

Reply:
merged